### ISSUE TITLE
"Preparing to teach" is fairly high-level workshop planning -- we may be missing a basic "how to prepare to the non-live-coding-bits"

### ISSUE THREAD

**Issue by Denubis** [carpentries/instructor-training#1226]
When I was preparing https://carpentries.github.io/instructor-training/15-lesson-study/index.html the tone seems to have moved towards workshop-organisation. Planning what exercises will fit learners, a discussion of learning objectives, and that sort of excellent "lead instructory" stuff.

What I find that we might be missing is pragmatic public-speaking/teaching material for "Well, we've told you to teach an episode, now get to it!" Doing a quick search on "reading" for example, I don't think we emphasise to "don't just read the lesson!" (or ever come out and say it) before they get to a teaching demo.

In the context of reverse instructional design, where we are explaining to support the formatives to support the goals, a discussion of good pedagogic practices **in relation to the texts** could be very useful. Participants may have little public speaking experience, or may be used to teaching from slides and adding contextualisation of "Well, you have this printed lesson in front of you, how do you go from that to teaching in front of a workshop, and how do you support your co-instructors doing same" could be a useful addition to this lesson. Looking at today's summary of changes (sorry for ducking out early) there does seem to be a deemphasis on the theoretical stuff, so this could be one way to take this page. 

Pragmatically, to me, this page needs to answer three questions:
1. Well, I printed out the text of a lesson, now what?
2. A lesson has all the exercises at the end (https://swcarpentry.github.io/sql-novice-survey/01-select/index.html), now what?
3. How do I adjust these stock lessons to my specific group?

I think we do best at number three right now. Not sure where to go from here though.

**Comment by ragamouf** [comment]
I am so here for this! Thank you for raising this issue! I particularly like how you've framed this issue with these 3 questions, and I will concentrate on 1 and 2.

For (1) ie now what? there is an opportunity to hark back to Ep 05 on concept maps, and run an exercise following the discussion for learners to map out (or detail in nested lists) the process of going from page to stage. What about an exercise in creating a timeline to delivery, then translated into a series of calendar slots. Not to be too prescriptive, but enough detail to understand how one might balance teaching prep around existing work and other things like life and sleep. 

(2) is a great case in point for instructors working out where to insert the exercises in the flow of teaching. Could be a good breakout room group activity using anything that allows for virtual moveable stickies like Google Jamboard (I know it's a terrible name, but a useful tool). 

How do these ideas sit with you?

**Comment by Denubis** [comment]
To move to a brief meta-discussion, I think this is a great time to use reverse instructional design :) And we also need to figure out what our time budget is for this "extra content" because it'll be competing with other content, especially as I know this chapter is in the middle of a revision.

@ragamouf Perhaps the place to start is to articulate learning objectives? You seem to have a nuanced understanding of the difficulties here, so it'd be interesting to see what you think good learning objectives would be in this context.

@karenword Any thoughts on parameters for exercise design or proposed episode length? (Obviously without obligation to accept, but it would be pointless to do too much work.)

**Comment by ragamouf** [comment]
Re: reverse instructional design: I would really like to see your workshop with @dvanic to be on offer as a regular bonus module for instructors preparing to teach - especially when it's been ~12 months since their instructor training. Did I zoom out too much?

I'm signed up for teaching instructor training in June, so I'm pretty happy to work on this as a way to solve (for myself!) how long to allow for prep for teaching. Am I agreeing to apply reverse instructional design to the reverse instructional design component of the lesson?

**Comment by Denubis** [comment]
Nah. Two different challenges. For June, budget 15-20 minutes on "how to prepare an episode" on the first day so they know what they need to do for the second day. (And general public speaking tips)

For meta^3... uh... ::shrug:: And for our workshop as bonus module, ask us for anything* but time.

*anything includes: the null set, end of list. It's a pretty restrictive anything right now.

**Comment by brownsarahm** [comment]
Karen has been working on this episode in the revision. You can see the the work in progress in the [data lessons repository](https://github.com/data-lessons/instructor-training/pull/38).

I love the idea of getting these points in the curriculum formally. I often end up answering a lot of trainee questions that address these points during the day 2 question list, after the second teaching practice, and during managing a diverse classroom, but not always.

**Comment by karenword** [comment]
I have no idea how we will fit any of this in... yet. :) I am not sure that I have a very concrete understanding of exactly what you are proposing -- I think there is a lot to unpack in "now what?" 

My suggestion is that you aim for a very minimalist approach at first. What could you do with one 10 minute activity? Where could we fit in angles of this on other existing content?

Revisions for this section are far from finished. I expect to be merging the linked PR soon (in data-lessons) and plan substantial additional work in a next PR.

**Comment by karenword** [comment]
phew! There is a lot packed in here! I think I have a better idea about where you are going, though. Thank you :)

The first activity is a fairly neat package that can probably find a home somewhere in the updates. 

The second item is more complicated, and I think there is a next step in deciding how to structure conversation and consensus around recommendations/ specific learning objectives here. The if-then situations are a challenge. Defining practices runs the risk of being excessively prescriptive. Nevertheless, I agree that we do need to provide more support here.

Maybe we bring this to a Trainer meeting, with some structure?

**Comment by dvanic** [comment]
Oof - lots to unpack, and I've got more. So I think something that really came up in the Q&A of our instructor training, that is useful as something we actually do in real life is - going from an episode of a lesson as written to something we actually deliver, i.e. the  question above: "An episode has all the exercises at the end/interspersed, now what?". This could be a combination of in-class and homework activities. The in-class, to add to Brian:

> In your groups, discuss for 15 minutes:
> As a group, find one of the exercises in the episodes you have prepared and identify everything a complete novice attending your workshop needs to understand before being able to accomplish the exercise.
> 
> - What do they need to know?
> - What do they already know from prior lessons?
> - What do you need to teach for them to be able to achieve the exercise?
> - We will discuss at some length as a workshop.

Also:

- What (1) new links between concepts and (2) skills will they develop as a result of this exercise?
- What links and skills will they practice? 

***

This would then build into the (real-world) activity of exercise-picking for an episode, which I would give as homework to be discussed the next day (works better for distributed online than canonical 2 day in person, I know - (meta: probably one to omit from the in-person)).

Basically, look at an episode from a lesson (ideally the same one you'll eventually use for the teaching demo). If multiple people are using the same lesson - use the same episode for more teaching gains.

Following the mapping procedure like what we used in the session, try to identify which combination of exercises you would use to teach the module, given that you will not be able to use all of them, and you want to achieve learning across all of the episode's key objectives and points. For each exercise on the page, assign it one of four ranks: "definitely skipping", "nice to have", "skip if running out of time", "essential". (IRL we usually end up with only the essentials, but our novice instructors don't know that! - I think it's a key lesson most instructors I've worked with learn the hard way 😿 ). 

The next day, have learners discuss their choices and logic in groups - hopefully some people working on the same episode will choose different exercises, which will make it even more fun! (and even more realistic).

Because (as y'all know) while I love reverse design and other learning _theory_, it's really only when we use those ideas to reflect on and shape our teaching practice do we truly understand it, and the beauty of the theory shines through...

**Comment by Denubis** [comment]
@dvanic I really like those updates to my exercise 1, and I think they might prove the necessary linchpin that would allow this to substitute for concept maps.

**Comment by Denubis** [comment]
@brownsarahm Can we drop this into two discussions over in trainers?
